internal/backends/selector/lib.rs

```diff
@@ -38,7 +38,7 @@ cfg_if::cfg_if! {
             let backend_config = std::env::var("SLINT_BACKEND").unwrap_or_default();
 
             let backend_config = backend_config.to_lowercase();
-            let (event_loop, _renderer) = backend_config.split_once('-').unwrap_or_else(|| match backend_config.as_str() {
+            let (event_loop, _renderer) = backend_config.split_once('-').unwrap_or(match backend_config.as_str() {
```

Comment:
Wouldn't it be better to stay with lazy evaluation instead of running the match every time?

Reply:
> OTOH creating a closure also has a cost

It shouldn't have a cost.  Or what kind of cost are you thinking of? Compile-time cost?